src/ai/backend/manager/config.py

```diff
@@ -288,6 +288,7 @@
         t.Key('hide-agents', default=False): t.Bool,
         t.Key('importer-image', default='lablup/importer:manylinux2010'): t.String,
         t.Key('max-wsmsg-size', default=16 * (2**20)): t.ToInt,  # default: 16 MiB
+        t.Key('aiomonitor-port', default=50001): t.Int,
```

Comment:
You could add the min-max constraints to `t.Int`: `t.Int[1:65535]` to specify the exact range of possible TCP port values. Pleaes update on the lablup/backend.ai-agent#295 as well!